gateway/schema/semver_test.go

```
@@ -0,0 +1,510 @@
+package schema
+
+import (
+	"testing"
+
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gci)